blockscout-ens/types/.gitignore

```diff
@@ -1,2 +1,2 @@
-/node-modules
+/node_modules
```

Comment:
Can we add node_modules to the root `.gitignore` instead to be ignored recursively?

Reply:
Currently there are 17 occurrences of "node_modules" in the project's `.gitignore` files. Should I remove them all in favor of the root file?

<img width="391" alt="image" src="https://github.com/blockscout/blockscout-rs/assets/22130104/f0160d9a-7dfa-4354-9488-1fee91172fb6">

Reply:
I think so

Reply:
Done ✅